cli/src/cast.rs

```diff
@@ -684,6 +658,14 @@ async fn main() -> eyre::Result<()> {
             println!("0x{}", hex::encode(selector));
         }
         Subcommands::FindBlock(cmd) => cmd.run()?.await?,
+        Subcommands::Estimate { to, 
+                                sig,
+                                args,
+                                value,
+                                eth,
+                                command } => {
```

Comment:
can we make this a stand alone struct? (`EstimateArgs`)

this way we don't need the verbose `fn run` and can make this a function of that struct instead `fn run(&self)` for `EstimateArgs`